packages/feedback/src/util/sendFeedbackRequest.ts

```diff
@@ -1,113 +1,41 @@
 import { getCurrentHub } from '@sentry/core';
-import { dsnToString } from '@sentry/utils';
+import type { FeedbackEvent } from '@sentry/types';
+import { uuid4 } from '@sentry/utils';
 
 import type { SendFeedbackData } from '../types';
-import { prepareFeedbackEvent } from './prepareFeedbackEvent';
 
 /**
  * Send feedback using `fetch()`
  */
-export async function sendFeedbackRequest({
+export function sendFeedbackRequest({
   feedback: { message, email, name, replay_id, url },
-}: SendFeedbackData): Promise<Response | null> {
+}: SendFeedbackData): string | undefined {
   const hub = getCurrentHub();
-
-  if (!hub) {
-    return null;
-  }
-
   const client = hub.getClient();
   const scope = hub.getScope();
-  const transport = client && client.getTransport();
-  const dsn = client && client.getDsn();
 
-  if (!client || !transport || !dsn) {
-    return null;
+  if (!client) {
+    return;
   }
 
-  const baseEvent = {
-    feedback: {
-      contact_email: email,
-      name,
-      message,
-      replay_id,
-      url,
+  const baseEvent: FeedbackEvent = {
+    contexts: {
+      feedback: {
+        contact_email: email,
+        name,
+        message,
+        replay_id,
+        url,
+      },
     },
-    // type: 'feedback_event',
+    type: 'feedback',
   };
 
-  const feedbackEvent = await prepareFeedbackEvent({
+  return client.captureEvent(
+    baseEvent,
+    {
+      event_id: uuid4(),
+    },
```

Comment:
What are the trade offs for using `captureEvent` @mydea? We haven't really talked about any hooks that would be needed, but could be a possibility down the road. This would save us from having a lot of custom code, but I don't know how to find out if `captureEvent` fails so that we can show the user an error.

Reply:
I think the core thing is for it to go through event processors etc. Which we have to think about if we _want_ this even 🤔